content-services/latest/admin/import-transfer.md

```diff
@@ -314,7 +314,7 @@ It supports sync mode transfer, so it can also be used by the replication servic
 
 The File System Transfer Receiver is delivered as a compressed zip file.
 
-1. Download the following file from the Alfresco Support Portal:
+1. Download the following file from the Support Portal:
 
     `alfresco-file-transfer-receiver-7.0.x.zip`
```

Comment:
Should this be changed at this point? 
The file I am working with with AGS is Alfresco-Content-Services-Distribution-7.1.0.1.zip

Reply:
Not for this artefact - it's still FSTR v7.0.0